Code/GraphMol/DistGeomHelpers/Wrap/rdDistGeom.cpp

```diff
@@ -509,6 +509,9 @@ BOOST_PYTHON_MODULE(rdDistGeom) {
       .def_readwrite("optimizerForceTol", &PyEmbedParameters::optimizerForceTol,
                      "the tolerance to be used during the distance-geometry "
                      "force field minimization")
+      .def_readwrite("basinThresh", &PyEmbedParameters::basinThresh,
+                     "set the basin threshold for the DGeom force field. "
+                     "This shouldn't be changed in client code!")
```

Comment:
I know it's in the C++ code, but after thinking about it I think this warning can be removed.

It would be kind of dumb to expose an option along with a comment saying "don't use this!" :-)